dist/dropdown/ds6/dropdown.css

```diff
@@ -37,15 +44,16 @@ span.fake-menu__items {
   margin: 0;
 }
 div.menu__item[role^="menuitem"],
+div.listbox__option[role^="option"],
 a.fake-menu__item,
 button.fake-menu__item {
-  background-color: white;
+  background-color: #ffffff;
   box-sizing: border-box;
+  color: #111820;
```

Comment:
Is this a change in color?

Reply:
I used the spec you gave me as reference.